contracts/utils/Address.sol

```diff
@@ -34,6 +34,18 @@ library Address {
         return size > 0;
     }
 
+    /**
+     * @dev Returns true if and only if the function is running in the constructor
+     */
+    function isConstructor() internal view returns (bool) {
```

Comment:
This is my only issue with this PR. This function is not really about addresses, it's more about the "lifecycle" of a contract, and it happens to use `Address.isContract` in its implementation.

I would prefer to keep `isConstructor` where it was originally, but using `Address.isContract` instead of the assembly it was using before.